### ISSUE TITLE
[checkbox group] support for arrow keys navigation

### ISSUE THREAD

**Issue by alexis-dorosz** [mui/base-ui#2012]
# Feature request

## Summary

Checkbox Group current does not have arrow keys keyboard navigation, unlike in radix-theme

## Examples in other libraries

Radix-theme checkbox group https://www.radix-ui.com/themes/docs/components/checkbox-group

## Motivation

More intuitive keyboard navigation within a group of related elements

**Comment by atomiks** [comment]
Checkbox groups don't use composite arrow key navigation intentionally: https://github.com/mui/base-ui/issues/1889#issuecomment-2862277561
